More Bingo Stuff

FleurBleue101 opened the pull request:
Added:

Grazer Dummies: 5, 10, 15 and All.
Blazing Suns: 3, 6 and 9.
Ancient Vessels: 3, 6, 9 and All.
Bandit Camps: 3, 4, 5, 6 and All.
Cauldrons: 1, 2 and 3.
Tallnecks: 2, 3 and 4.
Sidequests: ShieldWeaver, Traitor's Bounty and Queen's Gambit.

AltinCloak reviewed (commented):
I think that for grazer dummies it should be half and all. I already did Ancient Vessels already and I think that the other activity should be more specific, "Complete Cauldron SIGMA", as well as maybe do half and all.

FleurBleue101 commented:
> I think that for grazer dummies it should be half and all. I already did Ancient Vessels already and I think that the other activity should be more specific, "Complete Cauldron SIGMA", as well as maybe do half and all.

Changed the grazer dummies to "12 Grazer Dummies" and "All Grazer Dummies", and also changed the cauldrons to be each cauldron specifically, along with All Cauldrons, also removed my version of ancient vessels
